operating/manage-user-access/configure-ocp-oauth.adoc

```diff
@@ -0,0 +1,34 @@
+[id="configure-ocp-oauth"]
+= Configuring {ocp} OAuth server as an identity provider
+include::modules/common-attributes.adoc[]
+:context: configure-ocp-oauth
+
+toc::[]
+
+[role="_abstract"]
+{ocp} includes a built-in OAuth server and you can use it as an authentication provider for {product-title} (RHACS).
+
+[CAUTION]
+====
+The {ocp} OAuth server integration only works when you install {product-title} by using the `roxctl` CLI on {ocp}. It doesn't work when you install {product-title} by using Helm or the {product-title} Operator.
+====
+
+[IMPORTANT]
+====
+* To disable the {ocp} OAuth server integration, set the `ROX_ENABLE_OPENSHIFT_AUTH` environment variable as `false` in Central:
++
+[source,terminal]
+----
+$ oc -n stackrox set env deploy/central ROX_ENABLE_OPENSHIFT_AUTH=false
+----
++
+When you disable the {ocp} OAuth server integration, all the previously configured {ocp} OAuth providers stop working and they are not listed as a login option. The UI still shows the option to configure {ocp} OAuth server, but the integration will not work.
+* Integration with {ocp} OAuth server is only available for {ocp} 4.1 and newer.
+* If you use a custom TLS certificate for {ocp} OAuth server, you must add the certificate as a trusted root CA or as a Kubernetes secret `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt`. Otherwise, Central will not establish a TLS connection with the {ocp} OAuth server.
+====
```

Comment:
In general, we try to avoid stacking admonitions. I am guessing there's not an easy way out of this one, though.